### PR TITLE
busybox: update to version 1.24.2

### PR DESCRIPTION
**Pull request by esben**
Without this, I seem to be unable to decompress files larger than ~1036 bytes!

### CODE REVIEW

**Comment by skny-prevas** [comment]
Why not bump the busybox recipe to version 0.25 where this patch is included?

**Comment by sknsean** [comment]
Why not bump the busybox recipe to version 0.25 where this patch is included?

**Comment by esben** [comment]
Busybox 1.25.0 is labelled "unstable", so that is a bit early.
An upgrade to 1.24.2 would be fine though, but I just made this fix locally, so thought it would be better to share it than just sit on it.

File: recipes/busybox/busybox-1.24.2/default_defconfig [hunk 1]
```diff
@@ -0,0 +1,1073 @@
+#
+# Automatically generated make config: don't edit
+# Busybox version: 1.24.1
```
Comment (sknsean) [inline recipes/busybox/busybox-1.24.2/default_defconfig]: Nit picking :-)
Version number does not match the recipe version

Comment (esben) [inline recipes/busybox/busybox-1.24.2/default_defconfig]: Nit picking contest accepted ;)
The comment is correct, it was automatically generated with busybox version 1.24.1.
Updating the comment would therefore be falsifying the evidence.

**Comment by esben** [comment]
I would like to have this included in 6.1.0.

**Pull request merged**